.travis.yml

```diff
@@ -1,22 +1,41 @@
+sudo: false
 language: python
 python:
   - "2.7"
 # command to install wxPython using apt
-before_install:
-  - sudo apt-get update -qq
-  - sudo apt-get install -qq python-wxgtk2.8
+# before_install:
+#   - sudo apt-get update -qq
+#   - sudo apt-get install -qq python-wxgtk2.8
```

Comment:
Let's delete these 3 lines instead of commenting them out, assuming I'm understanding correctly that we don't need them now.